libs/utils/src/environment_variable.ts

```diff
@@ -130,6 +134,12 @@ export function getBooleanEnvVarConfig(
         allowInProduction: false,
         autoEnableInDevelopment: false,
       };
+    case BooleanEnvironmentVariableName.ENABLE_DEV_DOCK:
+      return {
+        name,
+        allowInProduction: false,
+        autoEnableInDevelopment: true,
+      };
```

Comment:
It may be worth a note that `USE_MOCK_CARDS` must also be enabled for the dev dock card mocking to work. There's also a case for auto-enabling that env. var in development.

Reply:
Decided to move forward with disabling and showing a message